hmda/src/test/scala/hmda/validation/rules/lar/validity/common/V710Spec.scala

```diff
@@ -11,8 +11,6 @@ class V710Spec extends LarEditCheckSpec {
 
   property("If credit is reported exempt, all fields must be reported exempt") {
     forAll(larGen) { lar =>
-      lar.mustPass
```

Comment:
why was this removed from the test?

Reply:
That line tests that any randomly generated LAR should pass that edit. Which should never be the case when an edit is run. Only a subset of LARs should always pass or always fail this edit, which is tested below in that file. I am not sure why that line was added.